libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.stories.ts

```diff
@@ -25,15 +25,15 @@ storiesOf('Arrow Variation', module)
       template: `
       <div style="${containerStyle}">
         <div style="${arrowDivStyle}">
-          <gor-arrow-variation size='sm' [value]="1"></gor-arrow-variation>
-          Smaill
+          <gor-arrow-variation size='sm' [value]="1" [printing]="false"></gor-arrow-variation>
```

Comment:
After defining a default value for `_printing` you won't need to pass it here, so remember to update this as well.